README.md

```diff
@@ -74,4 +74,30 @@ $ poetry run pytest
 
 although if one only wants to launch the end-to-end tests add in `test_e2e` at the end of the command, or `tests` if one only wants to launch unit and integration tests.
 
-NOTE: For the End to End Tests to function, one needs to have `geckodriver.exe` in the root folder and have Mozilla Firefox installed in the Computer.
\ No newline at end of file
+NOTE: For the End to End Tests to function, one needs to have `geckodriver.exe` in the root folder and have Mozilla Firefox installed in the Computer.
+
+## Running the App (Docker instructions)
+
+Assuming one has already installed Docker and tested it works to get the app running, first build the image by running:
+
+```bash
+$ docker build --target <stage> --tag 
+```
+
+Where ```<stage>``` is either ```base```, ```development```, or ```production```, and ```<tag>``` is a tag of your choosing 
+
+If just wanting to run the image on production, run:
+
+```bash
+$ docker run --env-file .env  -p 8080:8000 todo-app:<production-tag> 
```

Comment:
Your tag here makes some assumptions about the tag that the user applied above, `todo-app:<production-tag>` vs `<tag> - might be nice to just keep this consistent

Reply:
Not sure what you mean by that, sorry